src/core/utils/internal/parseGraphQLRequest.ts

```diff
@@ -160,6 +160,7 @@ async function getGraphQLInput(request: Request): Promise<GraphQLInput | null> {
           variables,
         }
       }
+      return null
```

Comment:
Missing fallthrough lint warning (with new rules). 

Happy to move this to an ignore and then address this separately